lib/assets/javascripts/turbograft/response.coffee

```diff
@@ -23,12 +23,6 @@ class TurboGraft.Response
     else
       throw new Error("Error encountered for XHR Response: #{this}")
 
-  redirectedToNewUrl: () ->
-    Boolean(
-      @redirectedTo &&
```

Comment:
Does `@redirectedTo` need to be in class scope, now?

Reply:
good point, it can just be an implementation detail of `finalURL`